projects/gen-ai.md

```diff
@@ -0,0 +1,164 @@
+
+It helps GenAI observability community to
+
+- find solutions for new observability problems such as verbose events or correlation with async response evaluations
+- discuss implementation challenges
+- coordinate and prioritize common efforts
+
+### Goals, objectives, and requirements
+
+GenAI observability project is a new OTel cross-cutting SIG that intends to provide the GenAI community with a space to evolve and maintain GenAI related instrumentations in a **vendor-agnostic way**.
+
+It includes:
+
+- GenAI Semantic Conventions
+- Instrumentations for GenAI client libraries, frameworks, or other components.
+
+The eventual goal is to contribute instrumentation libraries to upstream components or to otel-contrib repos when upstream contributions are not possible.
+
+During initial development and experimentation, we may need to release libraries more frequently than it's possible for contrib and then may temporarily host instrumentation libraries in a `opentelemetry-{language}-genai` repository(ies).
+
+This would help GenAI observability vendors that will depend on these libraries to patch issues and adapt to external changes.
+
+## Deliverables
+
+Immediate term:
+- Ship OTel instrumentation libraries for OpenAI (or any other GenAI client) in Python and JS following existing conventions
+
+Middle term:
+- Ship OpenTelemetry (or native) instrumentations for popular GenAI client libraries in Python and JS covering chat calls
+- Evolve GenAI semantic conventions to cover other popular GenAI operations such as embeddings, image or audio generation
+
+As a result we should have feature parity with the instrumentations of existing GenAI Observability vendors for a set of client
+instrumentation libraries that all vendors can depend upon.
+
+Long term:
+- Implement instrumentations for GenAI orchestrators and GenAI frameworks for popular libraries in different languages
+- Evolve GenAI and other relevant conventions (DB) to cover complex multi-step scenarios such as RAG
+- Propose mature instrumentations to upstream libraries/frameworks
+
+## Staffing / Help Wanted
+
+The initial set of approvers for each language must include:
+
+- representatives of at least 2 different observability vendors that are active members of OTel GenAI SemConv WG
+- at least one existing approver or maintainer of the contrib, instrumentation or core OTel repo in the corresponding language
+
+**Project Leads:**
+
+- @drewby
+- @nirga
+
+**Sponsoring Members:**
+
+- @lmolkova
+- TBD
+
+**Engineers:**
+
+- **initial** maintainers (based on the existing membership in the corresponding repo)
+  - Python
+    - @lzchen
+    - @xrmx
+    - TBD
+  - JavaScript
+    - @trentm
+    - *looking for existing OTel JS maintainers for general JS & OTel feedback*
+    - TBD
+- **initial** JS and Python GenAI component/code owners (based on LLM Semantic Convention project involvement):
```

Comment:
Component/code owner isn't a formal concept in the opentelemetry project. In `opentelemetry-java-contrib`, the bar is fairly low:
- must be a member of the OpenTelemetry organization (required for github automation)
- must be responsive to issues / PRs
- must express interest in being a code owner

Not sure what the criteria is for other `*-contrib` repositories. It looks like `opentelemetry-collector-contrib` has [some documentation](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#becoming-a-code-owner) on the role, but it boils down to "the person is interested and the existing code owners / maintainers approve".

Can / should we broaden this group of code owners to the group of people that meet this loose criteria (member of Otel org, ready to respond to issues / PRs, interested)? In my experience with contrib repos, the problem has been underparticipation, not over participation.

Reply:
I agree with you, the point is that we're going to work in the existing repos and will follow their process. JS provides a great formal model of [component ownership](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/CONTRIBUTING.md#component-ownership).

In other repos I believe it's common to make people who contributed the library owners.

I realize that the list of maintainers and approvers in this proposal is misleading - there is no new status this project is about to provide - we'll just follow repo-specific practices. I remove it.